document-warehouse/get-document-schema.js

```diff
@@ -0,0 +1,64 @@
+/** Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+async function main(
+  projectNumber = 'YOUR_PROJECT_NUMBER',
+  location = 'YOUR_PROJECT_LOCATION',
+  documentSchemaId = 'YOUR_DOCUMENT_SCHEMA_ID'
+) {
+  // [START contentwarehouse_get_document_schema]
+
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   * const projectNumber = 'YOUR_PROJECT_NUMBER';
+   * const location = 'YOUR_PROJECT_LOCATION'; // Format is 'us' or 'eu'
+   * const schemaId = 'YOUR_DOCUMENT_SCHEMA_ID';
+   */
+
+  // Import from google cloud
+  const {DocumentSchemaServiceClient} =
+    require('@google-cloud/contentwarehouse').v1;
+
+  // Create service client
+  const serviceClient = new DocumentSchemaServiceClient();
+
+  // Get Document Schema
+  async function getDocumentSchema() {
+    // Initialize request argument(s)
+    const request = {};
+
+    // The full resource name of the location, e.g.:
+    // projects/{project_number}/locations/{location}/documentSchemas/{document_schema_id}
+    request.name = `projects/${projectNumber}/locations/${location}/documentSchemas/${documentSchemaId}`;
```

Comment:
Initiate the `name` parameter when creating the request object. 

Also, use the client library's types whenever possible.

And there should be a method to construct the documentschema full name, use it instead of hard-coding the string.